src/main.rs

```diff
@@ -267,6 +267,55 @@ struct Fixer {
     rb: char,
 }
 
+/// A fixing error.
+struct FixErr {
+    // The operation underway. Printed in an error message of the form "failed
+    // to <op> <filename>".
+    op: String,
```

Comment:
nit: With the exception of one call it seems to me that you're always `format!()`ing the string you store here. So why not have a function that takes the operation and the filename and calls `format!()` internally? It's just a suggestion to factor it out, the patch is fine as it is.

Reply:
I see quite a few calls to `fix_err()` that don't involve an argument, roughly half of them.